app/src/main/java/com/example/myapplication/vm/LoginViewModel.kt

```diff
@@ -69,9 +81,32 @@ class LoginViewModel(app: Application) : AndroidViewModel(app) {
         _validEmail.value = null
     }
 
+    fun resetResponse() {
```

Comment:
there is no need to reset the response

Reply:
Since it's a LiveData, every time its value gets updated, all of its observers will be notified, even if you set this value as null, it will notify the observers that a new value is awaiting (this null, for example).